sormas-api/src/main/java/de/symeda/sormas/api/person/PersonIndexDto.java

```diff
@@ -53,6 +54,7 @@
 	private String phone;
 	@SensitiveData
 	private String emailAddress;
+	private Date changeDate;
```

Comment:
I don't see this used anywhere, what's the reason for adding it?

Reply:
it's used as the default sorting and i must select it because of the distinct